src/components/tooltip/Tooltip.tsx

```diff
@@ -299,7 +299,7 @@ class Tooltip extends React.Component<TooltipProps, State> {
         }
 
         if (showTooltip) {
-            if (!isLabelMatchingTooltipText || childAriaLabel === undefined) {
+            if (!ariaHidden && (!isLabelMatchingTooltipText || childAriaLabel === undefined)) {
```

Comment:
do we need it on the `if` below as well for `aria-errormessage`? if so, we could add `ariaHidden` to the outer `if` statement:
```
if (showTooltip && !ariaHidden) {
    [...]
}
```

Reply:
@tjuanitas I'm not sure here. I'm not familiar with the bug that was the cause for the `aria-errormessage`

Reply:
gotcha but I think it might have a similar issue

my understanding is that if `ariaHidden` is true, then the tooltip will be hidden to screen readers and so the component using `aria-errormessage` (the label, button, etc) will reference a tooltip ID that is not visible/accessible